modules/cnf-about-virtual-routing-and-forwarding.adoc

```diff
@@ -0,0 +1,12 @@
+[id="cnf-about-virtual-routing-and-forwarding_{context}"]
+= About virtual routing and forwarding
+
+Virtual routing and forwarding (VRF) devices combined with IP rules provide the ability to create virtual routing and forwarding domains. VRF reduces the number of permissions needed by CNF, and provides increased visibility of the network topology of secondary networks. This is used to provide multi-tenancy functionality, for example, where each tenant has their own unique routing tables and requires different default gateways.
+
+Processes can bind a socket to the VRF device. Packets through the binded socket use the routing table associated with the VRF device. An important feature of VRF is that it impacts only OSI model layer 3 traffic and above so L2 tools such as LLDP are not affected. This allows higher priority IP rules such as policy based routing to take precedence over the VRF device rules directing specific traffic.
+
+The CNI plug-in that supports VRF is shipped as part of the CNI plugins container in {product-title}. the VRF CNI plug-in defines an isolated environment and creates network interfaces with unique routing tables. The VRF CNI plug-in can also define an optional VRF to put the interface into. Two connections to the container network interface coming from outside can belong to different VRFs with the same IP address.
+
+== Benefits of secondary networks for pods for telco operators
```

Comment:
Not sure worth mentioning this here. This doc section will be generic for openshift.

Reply:
OK, removed this.